src/components/common/ui/comment/Comment.tsx

```diff
@@ -0,0 +1,13 @@
+import {Box, Typography} from "@mui/material";
+import * as styles from "./Comment.styles";
+
+const Comment = ({text, semester, year}) => {
```

Comment:
Add interface for better types support, like that

`interface CommentProps{
text:string;
semester:string;
year:string;
}`

And u can add this interface as shown [here](https://github.com/fictadvisor/fictadvisor-web/blob/34c222fef0de24db83add29f16998e654d55f936/src/components/common/ui/alert-mui/Alert.tsx#LL39C25-L39C25)